### PR TITLE
Xrddev 842 localgroups add e2etests

### PR DESCRIPTION
**Pull request by olliru**
End 2 end tests for localgroups add function
Fixed client certificate and logout tests
Updated chrome driver and configuration

### CODE REVIEW

File: src/proxy-ui-api/frontend/tests/e2e/pages/ssMainPage.js [hunk 1]
```diff
@@ -15,35 +15,234 @@ var navigateCommands = {
...
 
-var clientTabCommands = {};
 
 module.exports = {
   url: process.env.VUE_DEV_SERVER_URL,
   commands: [navigateCommands],
   elements: {
     clientsTab: { 
-      selector: '//div[contains(@class, "v-tabs__bar")]//a[text()="clients"]', 
+      selector: '//div[contains(@class, "v-tabs-bar__content")]//a[text()="Clients"]',
```
Comment (TJaakkola) [inline src/proxy-ui-api/frontend/tests/e2e/pages/ssMainPage.js]: Depending on generated class-names can crash when the library generating the html is updated. This relates to the comment below. There seems to be several issues like this so I won't create more comments about those. 

No need to act on this as the situation may be getting better considering those test-data attributes?

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-details.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  tags: ['ss', 'clients', 'clientdetails'],
+  'Security server clients list': browser => {
+    const frontPage = browser.page.ssFrontPage();
+    const mainPage = browser.page.ssMainPage();
+    const clientsTab = mainPage.section.clientsTab;
+    const clientInfo = mainPage.section.clientInfo;
+    const clientDetails = clientInfo.section.details;
+    const certificatePopup = mainPage.section.certificatePopup;
+
+    // Open SUT and check that page is loaded
```
Comment (TJaakkola) [inline src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-details.js]: What is 'SUT'? Perhaps break it down instead :)

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-list.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    const frontPage = browser.page.ssFrontPage();
+    const mainPage = browser.page.ssMainPage();
+    const clientsTab = mainPage.section.clientsTab
+
+    // Open SUT and check that page is loaded
+    frontPage.navigate();
+    browser.waitForElementVisible('//*[@id="app"]');
+
+    // Enter valid credentials
+    frontPage
```
Comment (TJaakkola) [inline src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-list.js]: This block is used several times. Maybe refactor it inside a function?

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-list.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+      .signin();
+
+    // Switch to clients tab and verify
+    mainPage.openClientsTab();
+    browser.waitForElementVisible(clientsTab);
+
+    // Verify default sorting and list items
+    browser
+      .waitForElementVisible('//th[span[contains(text(),"Name")] and contains(@class, "active") and contains(@aria-sort, "ascending")]')
+      .waitForElementVisible('(//tbody/tr)[1]//span[contains(text(),"TestCom")]')
```
Comment (TJaakkola) [inline src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-list.js]: Would it make it easier to create a function that generates these nearly identical strings. Something like this:

```
const generateSSClientTableRow = (index, text) => {
  return `(//tbody/tr)[${index}]//span[contains(text(), "${text}")]`
}
```

and then call it:
`generateSSClientTableRow(1, "TestCom")`

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-logout.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  'Security server logout': browser => {
+    const frontPage = browser.page.ssFrontPage();
+    const mainPage = browser.page.ssMainPage();
+
+    // Navigate to app and check that the browser has loaded the page
+    frontPage.navigate();
+    browser.waitForElementVisible('//*[@id="app"]');
+
+    // Enter valid credentials
+    frontPage
```
Comment (TJaakkola) [inline src/proxy-ui-api/frontend/tests/e2e/specs/ss-logout.js]: Again this duplicated block that could be refactored to a function

**Comment by jansu76** [comment]
Changes request to this PR are handled elsewhere. Closing this to clean up https://niis.slack.com/archives/GGBHVCYSY/p1595333532008600?thread_ts=1590999066.002500&cid=GGBHVCYSY

**Pull request closed without merging**